src/control/lib/spdk/src/nvme_control.c

```diff
@@ -28,6 +28,12 @@
 #include "nvme_control.h"
 #include "nvme_control_common.h"
 
+struct wipe_sequence {
+	struct ns_entry	*ns_entry;
+	char		*buf;
```

Comment:
Since this struct has a scope of this whole file, the name of this buffer should be more descriptive. lba0_data or something like that.

Reply:
done